SignalServiceKit/src/Messages/OWSMessageManager.m

```diff
@@ -85,12 +86,22 @@ - (instancetype)initWithPrimaryStorage:(OWSPrimaryStorage *)primaryStorage
     _primaryStorage = primaryStorage;
     _dbConnection = primaryStorage.newDatabaseConnection;
     _incomingMessageFinder = [[OWSIncomingMessageFinder alloc] initWithPrimaryStorage:primaryStorage];
+    
+    /// Loki: Add observation for new session
```

Comment:
Triple slash is usually used for doc comments. Double slash is the standard for regular comments :)